ext/net/01_net.js

```diff
@@ -419,13 +419,17 @@ class Datagram {
 
 const listenOptionApiName = Symbol("listenOptionApiName");
 
-function listen(args) {
+function listen(args, checkUnstable = true) {
```

Comment:
This is a public API change?

Reply:
Yeah, we can't do it; we need to add another `op_net_listen_tcp` op that skips the unstable check. And we should stabilize the `reusePort` option in the next minor.

Reply:
This is `Deno.listen`